radix-engine-interface/src/blueprints/epoch_manager/invocations.rs

```diff
@@ -20,11 +20,40 @@ pub struct EpochManagerCreateInput {
     pub initial_configuration: EpochManagerInitialConfiguration,
 }
 
-#[derive(Debug, Eq, PartialEq, Sbor)]
+#[derive(Debug, Eq, PartialEq, ScryptoSbor, ManifestSbor)]
 pub struct EpochManagerInitialConfiguration {
     pub max_validators: u32,
     pub rounds_per_epoch: u64,
     pub num_unstake_epochs: u64,
+    pub total_emission_xrd_per_epoch: Decimal,
+    pub min_validator_reliability: Decimal,
+}
+
+impl EpochManagerInitialConfiguration {
```

Comment:
hmm, all items of this `EpochManagerInitialConfiguration` are mandatory, and they don't really have business-level default values (I mean: I assume that Engine does not want to define [or even suggest?] production values for `max_validators` etc. - at least it wasn't doing it so far).

for this ☝️ reason, I don't see a separate builder struct as that helpful (it would have to take all these fields in the constructor anyway).